patternlab/styleguide/source/_patterns/02-molecules/footer-links.twig

```diff
@@ -1,14 +1,14 @@
 <section class="ma__footer-links">
-  {% for item in footerLinks.items %}
-    <nav role="navigation" aria-labeledby="{{ item.id }}">
-      <h2 class="visually-hidden" id="{{ item.id }}">{{ item.heading }}</h2>
+  <nav role="navigation" aria-labeledby="footerMenu">

```

Comment:
`aria-labeledby` has typo.  Please change it to `aria-labelledby`.  It has 2 ls.